src/commands/scan.yml

```diff
@@ -94,7 +99,7 @@ steps:
         SNYK_INTEGRATION_NAME: CIRCLECI_ORB
```

Comment:
I can't leave a comment on the exact line but "run.name" just above this will need to change to be a bit more generic (or use the parameter if it's available).

Reply:
Is "Run Snyk" too generic? It really doesnt matter tbh, the job names not very important

Reply:
That's true, I don't mind a generic name either until a user flags it as a problem.

If it does become a problem, assuming we can use parameters, we can add a `name` parameter and let users override it like they can when they use `run` directly.